.cspell.yml

```diff
@@ -170,3 +170,5 @@ words:
   - WSGI
   - zend
   - zipkin
+  - otelsql
+  - sqlcommenter
```

Comment:
These should go in the page-local dictionary IMHO. WDYT @svrnm

Reply:
sqlcommenter might eventually land there, but yes, they should be page local, I missed that one, thanks!

@XSAM add those terms to the cspell:ignore in the front matter of your blog post, see existing blog posts how that works if you do not have one yet.